### DISCUSSION TITLE
Question about 41_snapshots-btrfs

### DISCUSSION THREAD

**Discussion by bkmo** [Antynea/grub-btrfs#290] [Q&A] [answered]
Just something that bothers me....
Why does this  get printed in the terminal when 41_snapshots-btrfs is run from the terminal? What can I do to suppress the output of CAT in the terminal?
```
if [ ! -e "${prefix}/grub-btrfs.cfg" ]; then
echo ""
else
submenu 'Arch Linux snapshots' {
    configfile "${prefix}/grub-btrfs.cfg"
}
fi

```

**Comment by bkmo** [comment, marked as answer]
Nevermind.....if I redirect it to  >  /dev/null  I do not get the output from cat.

**Reply by Schievel1** [reply]
This needs to be done to create the snapshots menu with `grub-mkconfig`, though. The grub scripts work by writing things to stdout. `grub-mkconfig` collects the output of the scripts and incorporates it into `grub.cfg`.
